ControlMeasurements/Models/MeasurementCategory.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+
+namespace ControlMeasurements.Models
+{
+    public class MeasurementCategory
+    {
+        public Guid Id { get; set; }
+        public PlaceType PlaceType { get; set; }
+        public MeasurementType MeasurementType { get; set; }
+        public int Value { get; set; }
+    }
+
+    public enum PlaceType
```

Comment:
Enumy mogą być w jednym pliku z klasa, ale zwyczajowo się je wydziela do osobnych plików (np w folderze Enums). Ale to tez kosmetyka

Reply:
Narazie zostawiłem je w jednym pliku bo miałem mały problem z nimi jak je dałem do innego folderu ;)